### ISSUE TITLE
Choropleth plot not working with MSA

### ISSUE THREAD

**Issue by ckonichipaulo** [cmu-delphi/covidcast#304]
`covidcast.plot_choropleth` with MSA level data is not working. The function works fine for me with `state` and `county`.

Here is my code:
```
datamsa = covidcast.signal("fb-survey",
                        "smoothed_cli",
                        date(2020, 8, 3),
                        date(2020, 8, 4),
                        geo_type="msa")
covidcast.plot_choropleth(datamsa)
plt.show()
```

And here is the error I'm getting:
```
There was a problem running this cell
DriverError /tmp/covidcast/shapefiles/msa/cb_2019_us_cbsa_5m.shp: No such file or directory ---------------------------------------------------------------------------
CPLE_OpenFailedError                      Traceback (most recent call last)
fiona/_shim.pyx in fiona._shim.gdal_open_vector()

fiona/_err.pyx in fiona._err.exc_wrap_pointer()

CPLE_OpenFailedError: /tmp/covidcast/shapefiles/msa/cb_2019_us_cbsa_5m.shp: No such file or directory

During handling of the above exception, another exception occurred:

DriverError                               Traceback (most recent call last)
<ipython-input-26-5c695b3eb263> in <module>
      4                         date(2020, 8, 4),
      5                         geo_type="msa")
----> 6 covidcast.plot_choropleth(datamsa)
      7 plt.show()

/tmp/covidcast/plotting.py in plot_choropleth(data, time_value, **kwargs)
     78     day_to_plot = time_value if time_value else max(data.time_value)
     79     day_data = data.loc[data.time_value == pd.to_datetime(day_to_plot), :]
---> 80     data_w_geo = get_geo_df(day_data)
     81 
     82     kwargs["vmin"] = kwargs.get("vmin", 0)

/tmp/covidcast/plotting.py in get_geo_df(data, geo_value_col, geo_type_col, join_type)
    169 
    170     shapefile_path = pkg_resources.resource_filename(__name__, SHAPEFILE_PATHS[geo_type])
--> 171     geo_info = gpd.read_file(shapefile_path)
    172 
    173     if geo_type == "state":

/tmp/geopandas/io/file.py in _read_file(filename, bbox, mask, rows, **kwargs)
     94 
     95     with fiona_env():
---> 96         with reader(path_or_bytes, **kwargs) as features:
     97 
     98             # In a future Fiona release the crs attribute of features will

/tmp/fiona/env.py in wrapper(*args, **kwargs)
    398     def wrapper(*args, **kwargs):
    399         if local._env:
--> 400             return f(*args, **kwargs)
    401         else:
    402             if isinstance(args[0], str):

/tmp/fiona/__init__.py in open(fp, mode, driver, schema, crs, encoding, layer, vfs, enabled_drivers, crs_wkt, **kwargs)
    255         if mode in ('a', 'r'):
    256             c = Collection(path, mode, driver=driver, encoding=encoding,
--> 257                            layer=layer, enabled_drivers=enabled_drivers, **kwargs)
    258         elif mode == 'w':
    259             if schema:

/tmp/fiona/collection.py in __init__(self, path, mode, driver, schema, crs, encoding, layer, vsi, archive, enabled_drivers, crs_wkt, ignore_fields, ignore_geometry, **kwargs)
    160             if self.mode == 'r':
    161                 self.session = Session()
--> 162                 self.session.start(self, **kwargs)
    163             elif self.mode in ('a', 'w'):
    164                 self.session = WritingSession()

fiona/ogrext.pyx in fiona.ogrext.Session.start()

fiona/_shim.pyx in fiona._shim.gdal_open_vector()

DriverError: /tmp/covidcast/shapefiles/msa/cb_2019_us_cbsa_5m.shp: No such file or directory
```

**Comment by chinandrew** [comment]
Thanks for reporting this, looking into it now.

**Comment by chinandrew** [comment]
Looks like it's a packaging issue with the MSA (and HRR) data files. A bit odd that the state and county worked fine though... Will work on a fix and push out a new version.

**Comment by ckonichipaulo** [comment]
Thank you so much for looking into it!

**Comment by chinandrew** [comment]
Fix should be in place and we'll be cutting a new release once we update and proofread some documentation in the next few days.

**Comment by capnrefsmmat** [comment]
This should be fixed in covidcast version 0.1.1, now available on PyPI. You can upgrade with

```
pip3 install --upgrade covidcast
```

(or the equivalent for Conda or another package manager, if you didn't use pip)

Let us know if it still doesn't work or anything else goes wrong.

**Comment by ckonichipaulo** [comment]
Thank you!!
